src/Link/Link.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, MouseEventHandler, ReactNode } from 'react'
+import styled, { css } from 'styled-components'
+import { darken } from 'polished'
+
+import { focusOutline } from '../utils/focusOutline'
+import { theme } from '../theme'
+import { Icon } from '../Icon'
+
+export type LinkProps = {
+  className?: string
+  href: string
+  onClick?: MouseEventHandler
+  openInNewTab?: boolean
+  children?: ReactNode
```

Comment:
Didn't we also had a download prop?

Reply:
Yes it was, I forgot to bring it up but from what I read `download` is useless for cross-browser file/content
It works only when the URL is from the same domain, knowing this, I wasn't sure if it made sense to have it in the component?

Reply:
yeah but in our case the download should only be applied to pdf that are usually uploaded to `assets.marshmallow.com` so we shouldn't have any issue with CORS.
If we wanna drop the `download` attribute then I think we should change the way `openInNewTab` and the icon works cause in home we will need to open the PDFs in a new tab but the icon of the link is different

This is what has been designed to open/download the IPID docs in home 
<img width="165" alt="Screenshot 2022-05-05 at 08 30 41" src="https://user-images.githubusercontent.com/1053476/166879521-3c9e951d-4655-4646-b1f4-97f8de8a3d9c.png">

Reply:
I thought most of our files were uploaded in a S3 bucket, my bad! I'll add it back